screener.config.js

```diff
@@ -28,15 +30,28 @@ const screenerConfig = {
   // screenshot every example in maximized mode
   states: glob
```

Comment:
please, introduce reference to the corresponding Screener docs file in the comment